pyvista/plotting/colors.py

```diff
@@ -875,6 +875,16 @@ def __hash__(self):  # pragma: no cover
         """Hash calculation."""
         return hash((self._red, self._green, self._blue, self._opacity))
 
+    def __getitem__(self, item):
+        """Support indexing for backward compatibility."""
+        if item < 0 or item > 3:
```

Comment:
Do we want to handle string keys? Will people try `color['red']`? If yes, just trying to index and catching errors might be better for users.

Reply:
Ah yes supporting string keys sounds good! What do you mean exactly by 'just tring to index and catching errors might be better for users'?

Reply:
Sorry, I was trying to be concise from mobile and overdid it :smile: What I meant was instead of checking integer index range, do `try: return self.float_rgba[i]` and print an informative error whenever that fails.

But actually supporting string keys (while preserving the original int behaviour) works too!